R/issues_descriptions.R

```diff
@@ -53,3 +63,19 @@ issues_descriptions_remotes <- function(meta) {
   meta$issue <- meta$issue | vapply(meta$remotes, length, integer(1L)) > 0L
   meta
 }
+
+issues_descriptions_version_conflict <- function(meta, repo) {
+  conflict <- vapply(
+    X = seq_len(nrow(meta)),
+    FUN = function(index) {
+      utils::compareVersion(
+        a = meta$version[index],
+        b = meta[[repo]][index]
+      ) < 0L
+    },
+    FUN.VALUE = logical(1L)
+  )
+  meta[[repo]][!conflict] <- NA_character_
```

Comment:
Is there any advantage to using `NA_character_` vs. `NA` (globally)?

Reply:
I like it for the sake of defensive programming. `NA` is actually a logical type, but the element is supposed to be a character. I can never predict when a difference in types will cause a subtle bug.